django_freeradius/utils.py

```diff
@@ -0,0 +1,39 @@
+from django.contrib.auth import get_user_model
+from django.core.files import File
+from django.template.loader import get_template
+from django.utils.crypto import get_random_string
+from xhtml2pdf import pisa
+
+
+def find_available_username(username, users_list):
+    User = get_user_model()
+    suffix = 1
+    tmp = username
+    names_list = map(lambda x: x.username, users_list)
+    while User.objects.filter(username=tmp).exists() or tmp in names_list:
+        tmp = '{}{}'.format(username, suffix)
+        suffix += 1
+    return tmp
+
+
+def prefix_generate_users(prefix, n, password_length):
+    users_list = []
+    user_password = []
+    User = get_user_model()
+    for i in range(n):
+        username = find_available_username(prefix, users_list)
+        password = get_random_string(length=password_length)
+        u = User(username=username)
+        u.set_password(password)
+        users_list.append(u)
+        user_password.append([username, password])
+    return users_list, user_password
+
+
+def generate_pdf(prefix, data):
+    template = get_template('django_freeradius/prefix_pdf.html')
```

Comment:
let's make this configurable, please add a setting which defaults to this value, explain that the value can be changed to an absolute path which should point to a Django template (don't forget to add a couple of lines of docs).